packages/cli/test/functional/test_site/index.md

```diff
@@ -71,6 +71,10 @@ Page Variable with {{ global_variable }}
 
 **Test Page Variable and Included Variable Integrations**
 
+{% set outerNunjucksVariable %}
```

Comment:
Note: adding this because in `testPageVariablesInInclude.md` below we will test if this outer variable is leaking into the inner page.